sim/midas/src/main/scala/midas/models/dram/FASEDMemoryTimingModel.scala

```diff
@@ -552,9 +552,9 @@ class FASEDMemoryTimingModel(completeConfig: CompleteConfig, hostParams: Paramet
     // Accepts an elaborated memory model and generates a runtime configuration for it
     private def emitSettings(fileName: String, settings: Seq[(String, String)])(implicit p: Parameters): Unit = {
       val file = new File(p(OutputDir), fileName)
-      val writer = new FileWriter(file)
+      val writer = new FileWriter(file, wId != 0)
```

Comment:
i'm asusming this is to denote appending to an existing file --> can you include the kwarg if it has a good name

Reply:
This is a java class, so  I don't think keyword arguments work.

Reply:
Huh, the more you know.